doc/chomp_interface/chomp_interface_tutorial.rst

```diff
@@ -49,10 +49,81 @@ Using CHOMP with Your Robot
       <arg name="debug" value="$(arg debug)"/>
       <arg name="planner" value="chomp" />
     </include>
-#. Open the ``move_group.launch`` file in your ``<robot_moveit_config>/launch/`` folder and make two changes. First, add ``<arg name="planner" default="ompl" />`` just under the ``<launch>`` tag and second, within the ``<include ns="move_group">`` tag replace ``<arg name="pipeline" value="ompl" />`` with ``<arg name="pipeline" value="$(arg planner)" />``.
+#. Open the ``move_group.launch`` file in your ``<robot_moveit_config>/launch/`` folder and make two changes.
+
+ 7.1. First, add ``<arg name="planner" default="ompl" />`` just under the ``<launch>`` tag, and,
+
+ 7.2. Second, within the ``<include ns="move_group">`` tag replace ``<arg name="pipeline" value="ompl" />`` with ``<arg name="pipeline" value="$(arg planner)" />``.
 
 Running the Demo
 ----------------
 If you have the ``panda_moveit_config`` from the `ros-planning/panda_moveit_config <https://github.com/ros-planning/panda_moveit_config>`_ repository you should be able to simply run the demo: ::
 
-  roslaunch panda_moveit_config demo_chomp.launch
\ No newline at end of file
+  roslaunch panda_moveit_config demo_chomp.launch
+
+Testing CHOMP with Obstacles in the Scene
+-----------------------------------------
+To test CHOMP in an evironment with obstacles, you can run any of the sample python scripts 
+ 
+ - `collision_scene_test_1.py <./scripts/collision_scene_test_1.py>`_ or 
+ - `collision_scene_test_2.py <./scripts/collision_scene_test_2.py>`_).
+
+ The first scripts creates a complex scene with four ostacles. The second script creates a simple environment with one obstacle. One can change the position/size of the obstacles to change the scene. 
```

Comment:
remove space at start of sentence

Reply:
removed the space at the start of the sentence. "The first scripts...."